corehq/apps/export/views.py

```diff
@@ -172,6 +172,12 @@ def _get_form_exports_by_domain(domain, has_deid_permissions):
     return _get_saved_exports(domain, has_deid_permissions, old_exports_getter, new_exports_getter)
 
 
+def get_export_count_by_domain(domain):
```

Comment:
Weird that this is in views.py - seems like this and `_get_whatever_exports_by_domain` should all move to dbaccessors.

Reply:
Nevermind, I'm just going to do this in https://github.com/dimagi/commcare-hq/pull/18606